### ISSUE TITLE
Avoid using private variables outside their classes

### ISSUE THREAD

**Issue by mossroy** [kiwix/kiwix-js#75]
Following #72, there are several parts of the code where we directly use private variables

**Comment by kelson42** [comment]
@Jaifroid Do we know if this is still the case? Detailing the way to detect and fix this kind of problems might help to close this old ticket fearly quickly?

**Comment by Jaifroid** [comment]
Yes, for example we use a kludge to access `_file.files` from `selectedArchive`. The ZimFile object gets attached to the ZimArchive object, for convenience. But this is quite deeply embedded in the code (was like that from very early on, probably from Evopedia days), and I think it might be "too late" to undo it without extensive code redesign that is not warranted if we manage to shift largely to libzim for modern browsers. Would you recommend closing without fixing?
